### PR TITLE
Update mediapicker article with an example

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [x] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

File: 13/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/media-picker-3.md [hunk 1]
```diff
@@ -4,7 +4,10 @@
 
 `Returns: IEnumerable<MediaWithCrops>` or `MediaWithCrops`
 
-This property editors returns a single `MediaWithCrops` item if the "Pick multiple items" Data Type setting is disabled or a collection if it is enabled.
+This property editors returns:
```
Comment (sofietoft) [inline 13/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/media-picker-3.md:7]: To make it a little clearer that it's either or.

```suggestion
This property editors returns one if the following:
```

File: 13/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/media-picker-3.md [hunk 2]
```diff
@@ -83,6 +86,23 @@ Global crops are configured on the Image Cropper property of the Image Media Typ
 }
 ```
 
+### Multiple enabled without Modelsbuilder to retrieve IEnumerable<IPublishedContent> data
```
Comment (sofietoft) [inline 13/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/media-picker-3.md:89]: Shouldn't it be after the "Multiple enabled without Modelsbuilder"?
Or, perhaps it should even be a sub section below that one, as it's still multiple enabled and without the Models Builder 🤔

**Review by sofietoft — COMMENTED** [review]
I've added comments to 13 only.
Apply them to all if you agree with the suggestions 🙌

**Comment by eshanrnh** [comment]
Thanks @sofietoft I've applied your suggestions 🙌

**Review by sofietoft — APPROVED** [review]
Looks great!

**Pull request merged**